com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -31,6 +31,19 @@ internal enum __RpcExecStage
             Client = 2
         }
 
+        private static void SetUpdateStage<T>(ref T param) where T: IHasUpdateStage
+        {
+            if (param.UpdateStage == NetworkUpdateStage.Unset)
+            {
+                param.UpdateStage = NetworkUpdateLoop.UpdateStage;
+
+                if (param.UpdateStage == NetworkUpdateStage.Initialization)
```

Comment:
I must be missing something...if we just set `param.UpdateStage` to `NetworkUpdateLoop.UpdateStage`, how could it be `NetworkUpdateStage.Initialization` unless there's some hidden behavior in the setter